src/components/Header/Header.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please don't disable the linter instead fix them manually, It will help you to write more clean code. Also, do this for the `Calculator` and `Quotes` components.